FogCreek/Glitch-Community
# Adhesive hero: fix broken homepage publishing

## Links
* Remix link: https://adhesive-hero.glitch.me/
* Issue-tracker link
* Specs/doc links

## Changes:
* fixed `saveHomeDataToFile`
* added `origin` back to `useGlobals` context
* made HomePreview do a hard reload after updating

## How To Test:
* log into https://adhesive-hero.glitch.me/
* https://adhesive-hero.glitch.me/ should show "magic sketchpad" as the first "fresh app"
* https://adhesive-hero.glitch.me/index/preview should show "powerslides" as the first "fresh app"
* clicking "publish" should redirect to the 'live' homepage and show "powerslides" as the first "fresh app"
